controllers/ibu_controller.go

```diff
@@ -142,7 +142,12 @@ func (r *ImageBasedUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	if isTransitionRequested(ibu) {
-		if validateStageTransition(ibu) {
+		var isAfterPivot bool
+		isAfterPivot, err = r.RPMOstreeClient.IsStaterootBooted(getStaterootName(ibu.Spec.SeedImageRef.Version))
```

Comment:
@donpenney I renamed the function and moved it to `rpmOstreeClient`. Do you think the variable name is okay?